File: openmedstack.sample/src/test/java/org/openmedstack/sample/TestCommand.java
package org.openmedstack.sample;

import org.openmedstack.commands.DomainCommand;

import java.time.Instant;

public class TestCommand extends DomainCommand {
    protected TestCommand(String aggregateId) {
        super(aggregateId, 1, Instant.now(), null);
    }
}

File: messaging.guice/src/test/kotlin/org/openmedstack/messaging/guice/TestCommand.kt
package org.openmedstack.messaging.guice

import org.openmedstack.commands.DomainCommand
import java.time.Instant

class TestCommand(aggregateId: String, version: Int, timeStamp: Instant, correlationId: String? = null) :
    DomainCommand(aggregateId, version, timeStamp, correlationId) {}